test/Tests/src/Data/Time/Date_Spec.enso

```diff
@@ -139,6 +139,22 @@ spec_with name create_new_date parse_date =
             date_1>=datetime . should_fail_with Type_Error_Data
             date_1==datetime . should_be_false
 
+        Test.specify "should create date before epoch start" <|
+            (create_new_date 100 1 2).year   . should_equal 100
+            (create_new_date 100 1 2).month  . should_equal 1
+            (create_new_date 100 1 2).day    . should_equal 2
+
+        Test.specify "Gregorian calendar related functionality should fail before epoch start" <|
```

Comment:
Maybe also worth adding a test for stuff _just after_ the epoch? To validate that _these do work_.